docs(readme): update readme (#293)
nervosnetwork/mercury: README.md

```shell
$ echo '{
    "id": 1234,
    "jsonrpc": "2.0",
    "method": "get_block_info",
    "params": [{
        "block_number": 10000, 
        "block_hash": null
    }]
}' \
| tr -d '\n' \
| curl -H 'content-type: application/json' -d @- https://Mercury-testnet.ckbapp.dev
```

#### Attention
Public servers do not guarantee high availability and high performance. 
If you want to use Mercury in a production project, please deploy and run Mercury by yourself.

### 2. Run Mercury Locally

- Step 1. Run a ckb node. If you already have a running one, skip this step.
  - run a [mainnet node](https://docs.nervos.org/docs/basics/guides/mainnet)
  - run a [testnet node](https://docs.nervos.org/docs/basics/guides/testnet)
- Step 2. Edit `mainnet_config.toml` or `testnet_config.toml` according to mainnet or testnet. These config files are located in `./devtools/config/`.
- Step 3. Download the latest version of Mercury from the [release page](https://github.com/nervosnetwork/Mercury/releases).
- Step 4. Run Mercury.
```shell
## mainnet
$ Mercury -c devtools/config/mainnet_config.toml run
## testnet
$ Mercury -c devtools/config/testnet_config.toml run
```

> #### Recommended Hardware

Reviewer: ```suggestion
#### Minimum Hardware
```
```suggestion
#### Recommended Hardware
```

> 2 Cores - 4G Memory - 50G Disk and above.

Author: ```suggestion
4 Cores - 8G Memory - 500G Disk and above.

If you use a standalone server to run the Postgre server, then 50G Disk is enough. 
```

    CKB_NETWORK: testnet
```

- Step 3. Build Mercury images from the Dockerfile.
```shell
$ docker build -t Mercury .
```

- Step 4. Run Mercury via docker.
```shell
$ docker-compose up -d
```
or

```shell
$ docker run -d -p 8116:8116 -v {user_config_path}:/app/devtools/config Mercury:latest
```

#### Recommended Hardware

> 2 Cores - 4G Memory - 50G Disk and above.

Author: ```suggestion
4 Cores - 8G Memory - 500G Disk and above.
```
